connect/transforms/src/main/java/org/apache/kafka/connect/transforms/SetSchemaMetadata.java

```diff
@@ -35,29 +37,34 @@
     private static final Logger log = LoggerFactory.getLogger(SetSchemaMetadata.class);
 
     public static final String OVERVIEW_DOC =
-            "Set the schema name, version or both on the record's key (<code>" + Key.class.getName() + "</code>)"
-                    + " or value (<code>" + Value.class.getName() + "</code>) schema.";
+            "Set the name, namespace and version on the schema of the record's key " +
+                    "(<code>" + Key.class.getName() + "</code>) or value " +
+                    "(<code>" + Value.class.getName() + "</code>).";
 
     private interface ConfigName {
         String SCHEMA_NAME = "schema.name";
+        String SCHEMA_NAMESPACE = "schema.namespace";
         String SCHEMA_VERSION = "schema.version";
     }
 
     public static final ConfigDef CONFIG_DEF = new ConfigDef()
             .define(ConfigName.SCHEMA_NAME, ConfigDef.Type.STRING, null, ConfigDef.Importance.HIGH, "Schema name to set.")
+            .define(ConfigName.SCHEMA_NAMESPACE, ConfigDef.Type.STRING, null, ConfigDef.Importance.HIGH, "Namespace to append to the schema name.")
```

Comment:
Nit: aren't we _prepending_ the namespace, rather than _appending_ the namespace?
```suggestion
            .define(ConfigName.SCHEMA_NAMESPACE, ConfigDef.Type.STRING, null, ConfigDef.Importance.HIGH, "Namespace to prepend to the schema name.")
```

Reply:
Or just setting the entire value. Not prepending or appending a value. Would we not be setting a non-existing "namespace" field in the avro schema? With our jdbc connector, it doesnt create the namespace field at all. Its only if the schema name contains a '.' that it produces one.